boefjes/katalogus/storage/memory.py

```diff
@@ -93,10 +96,19 @@ def __init__(
         self._organisation = organisation
 
     def get_by_id(self, plugin_id: str, repository_id: str, organisation_id: str) -> bool:
-        return self._data[plugin_id]
+        return self._data[f"{organisation_id}.{plugin_id}"]
+
+    def get_all_enabled(self, organisation_id: str) -> Dict[str, List[str]]:
```

Comment:
I was wondering why `enabled` can't be an argument for `get_all` or why not filter on the results of `get_all` instead

Reply:
There is no `get_all` in the interface at this point, but when we need more functionality we could of course refactor.

Reply:
Even then this method could be quite nice I think